### PR TITLE
SQL MI template with jumpbox 

### PR DESCRIPTION
**Pull request by srdan-bozovic-msft**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [X ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* Added new SQL MI template that along with SQL MI creates jumpbox with SSMS installed

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@srdan-bozovic-msft - let's move .ci_skip, sub should be whitelisted, we can manually verify the deployment - resourceGroup will be locked for 48 hours?

**Comment by srdan-bozovic-msft** [comment]
@bmoore-msft yes, automated cleanup wouldn't work. I'll remove .ci_skip and see what will happen

File: 201-sqlmi-new-vnet-w-jumbox/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+                        }
+                    ]
+                }
+            },
+            "resources": [
+                {
+                  "name": "SetupChocolatey",
+                  "type": "extensions",
+                  "location": "[parameters('location')]",
+                  "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 201-sqlmi-new-vnet-w-jumbox/azuredeploy.json]: update apiVersion

File: 201-sqlmi-new-vnet-w-jumbox/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+                    "displayName": "SetupChocolatey"
+                  },
+                  "properties": {
+                    "publisher": "Microsoft.Compute",
+                    "type": "CustomScriptExtension",
+                    "typeHandlerVersion": "1.7",
+                    "autoUpgradeMinorVersion": true,
+                    "settings": {
+                      "fileUris": [
+                        "https://raw.githubusercontent.com/srdan-bozovic-msft/azure-quickstart-templates/srbozovi-msft-sqlmi/201-sqlmi-new-vnet-w-jumbox/scripts/installSSMS.ps1"
```
Comment (bmoore-msft) [inline 201-sqlmi-new-vnet-w-jumbox/azuredeploy.json]: this needs to come from this repo: see [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

File: 201-sqlmi-new-vnet-w-jumbox/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "managedInstanceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "administratorLogin": {
+            "value": "cloudSA"
```
Comment (bmoore-msft) [inline 201-sqlmi-new-vnet-w-jumbox/azuredeploy.parameters.json]: GEN-UNIQUE

**Pull request merged**